### DISCUSSION TITLE
Meson can not find "libcurl" on Debian 12

### DISCUSSION THREAD

**Discussion by buhtz** [ximion/appstream#649] [Q&A] [answered]
I am stuck with building appstream.

Meson tells me about missing "libcurl" but "libcurl4" debian package is installed. There are a lot of other "libcurl" packages when I use "apt-cache search". Which one is needed on Debian 12?

This is my build command:

    Dependency "libcurl" not found, tried pkgconfig and cmake

**Comment by ximion** [comment, marked as answer]
Choose any curl flavour that you like.
For example for GnuTLS: `sudo apt install libcurl4-gnutls-dev`
